storagetransfer/quickstart.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Command-line sample that creates a one-time transfer from a Google Cloud
+Storage bucket to another.
+"""
+
+
+import argparse
+
+# [START storagetransfer_quickstart]
+from google.cloud import storage_transfer
+
+
+def create_one_time_transfer(project_id: str, source_bucket: str,
+                             sink_bucket: str):
+    """Creates a one-time transfer job."""
+
+    client = storage_transfer.StorageTransferServiceClient()
+
+    # The ID of the Google Cloud Platform Project that owns the job
+    # project_id = 'my-project-id'
+
+    # Google Cloud Storage source bucket name
+    # source_bucket = 'my-gcs-source-bucket'
+
+    # Google Cloud Storage destination bucket name. Should be a Nearline bucket
```

Comment:
Remove the "should be a nearline bucket" part, this is just a simple example not meant to demonstrate that actual use case. If we were demonstrating that use case in this example, we'd do the 30-day untouched condition